src/Sylius/Bundle/AdminBundle/Resources/views/AdminUser/_form.html.twig

```diff
@@ -26,6 +26,7 @@
                 <label>{{ 'sylius.ui.avatar'|trans }}</label>
                 {{ form_row(form.avatar, {'label': false}) }}
             </div>
+            <a href="{{ path('sylius_admin_admin_user_remove_avatar', {'id': app.request.attributes.get('id')}) }}" id="delete-button" class="ui icon red labeled button"><i class="icon trash"></i> {{ 'sylius.ui.delete'|trans }}</a>
```

Comment:
Shouldn't this be a form with a PUT method? With a link, we're using GET to update the administrator, which is not appropriate

Reply:
Btw, this button probably should not be displayed when admin has no avatar set